scss/_base_code.scss

```
@@ -1,5 +1,9 @@
 @import 'settings';
 
+$color-pre-bg: rgba($color-x-dark, 0.03);
+$digit-width: 1ch; // measured width of one character in the monospaced font
+$code-sidebar-width: calc(#{$sph-inner} + (4 * #{$digit-width})) !default;
```

if the 'digit width' uses `1ch` (which literary is 1 character width) I don't think we need separate variable for that.
```suggestion
$code-sidebar-width: calc(#{$sph-inner} + 4ch) !default; // width based on padding + 4 digits width
```

---

I think we should keep this, we position the number column negative left by the width of one character, I think it helps make that clear?

---

Ah, haven't noticed that we do use negative positioning. Sure thing.

---

Thats @lyubomir-popov that is what I was thinking. 

I believe its clearer to simplify the code like this to show its 4 characters width your calculating:
```scss
code-sidebar-width: calc(#{$sph-inner} + 4ch) !default;
```
Over this:
```scss
$digit-width: 1ch;
$code-sidebar-width: calc(#{$sph-inner} + (4 * #{$digit-width})) !default;
```